webhook/README.md

```diff
@@ -0,0 +1,101 @@
+## Knative Webhooks
+
+Knative provides infrastructure for authoring webhooks under
+`knative.dev/pkg/webhook` and has a few built-in helpers for certain
+common admission control scenarios.  The built-in admission controllers
+are:
+1. Resource validation and defaulting (builds around `apis.Validatable`
+  and `apis.Defaultable` under `knative.dev/pkg/apis`).
+2. ConfigMap validation, which builds around similar patterns from
+  `knative.dev/pkg/configmap` (in particular the `store` concept)
+
+To illustrate standing up the webhook, let's start with one of these
+built-in admission controllers and then talk about how you can write
+your own admission controller.
+
+## Standing up a Webhook from an Admission Controller
+
+We provide facilities in `knative.dev/pkg/injection/sharedmain` to try and
+eliminate much of the boilerplate involved in standing up a webhook. For this
+example we will show how to stand up the webhook using the built-in admission
+controller for validating and defaulting resources.
+
+The code to stand up such a webhook looks roughly like this:
+
+```
+// Create a function matching this signature to pass into sharedmain.
+func NewResourceAdmissionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+	return resourcesemantics.NewAdmissionController(ctx,
+		// Name of the resource webhook (created via yaml)
+		fmt.Sprintf("resources.webhook.%s.knative.dev", system.Namespace()),
```

Comment:
perhaps show the example with s "your-name-here" manner?

Reply:
this is literally copied our of `knative.dev/sample-controller` 🤷‍♂

Reply:
I know, but if this is a walkthrough it's useful to have all the items that need customization to be called out
🤷‍♂